driver-legacy/build.gradle

```diff
@@ -38,3 +38,12 @@ dependencies {
     testCompile project(':driver-core').sourceSets.test.output
     testCompile project(':driver-sync').sourceSets.test.output
 }
+
+jar {
```

Comment:
I don't think driver-legacy can be an OSGi module, since it contains classes in the `com.mongodb` package.  Since driver-core also does, it means `com.mongodb` is split across the two modules, and OSGi doesn't allow that.

Reply:
The issue that I cannot resolve at the moment is how to pass `bnd` arguments without the `jar` task so that the manifest will contain the same information as in the 3.x manifest.